src/stores/GuildEmojiStore.js

```diff
@@ -1,15 +1,15 @@
 const DataStore = require('./DataStore');
-const Emoji = require('../structures/Emoji');
+const GuildEmoji = require('../structures/GuildEmoji');
 const ReactionEmoji = require('../structures/ReactionEmoji');
 
 /**
  * Stores emojis.
```

Comment:
This is a bit obscure.. Isn't there a store for the main client? If there is, this should state that `it stores guild emojis` (unless client uses a collection in which case :P)